src/main/java/controller/Controller.java

```diff
@@ -1,131 +1,97 @@
 package controller;
 
-import static domain.MenuOption.CHANGE_ATTENDANCE;
-import static domain.MenuOption.CHECK_ATTENDANCE;
-import static domain.MenuOption.SHOW_ALERT_CREW;
-import static domain.MenuOption.SHOW_CREW_ATTENDANCES;
-import static domain.MenuOption.getMenuOption;
-import static domain.MenuOption.isExit;
-
 import domain.Attendance;
-import domain.Crew;
-import domain.CrewGroup;
+import domain.AttendanceBook;
+import domain.Attendances;
+import domain.AttendancesFile;
 import domain.MenuOption;
 import domain.Time;
+import java.nio.file.Path;
+import java.time.LocalDate;
 import java.time.LocalDateTime;
-import java.util.List;
-import service.CrewLoader;
-import service.DateValidator;
+import java.util.Map;
 import view.InputView;
 import view.OutputView;
-import view.dto.AlertCrewDto;
-import view.dto.AlertCrewsDto;
-import view.dto.AttendanceLogDto;
-import view.dto.ChangeAttendanceLogDto;
-import view.dto.CrewAttendancesDto;
 
 public class Controller {
-    private static final LocalDateTime TODAY = LocalDateTime.of(2024, 12, 18, 10, 0);
+    private static final LocalDate today = LocalDate.of(2024, 12, 18);
+    private static final Path path = Path.of("src", "main", "resources", "attendances.csv");
```

Comment:
```suggestion
    private static final LocalDate TODAY = LocalDate.of(2024, 12, 18);
    private static final Path PATH = Path.of("src", "main", "resources", "attendances.csv");
```

상수 네이밍 컨벤션은 대문자 + 스네크케이스입니다!

Reply:
헉 넵 감사합니다 😃